### PR TITLE
README: Include instructions for how to join conda-forge/staged-recipes team

### PR DESCRIPTION
**Pull request by smithsp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,3 +117,6 @@ git rebase upstream/master
 git push -f
 ```
 
+### 12. What is the procedure to become a member of the @conda-forge/staged-recipes team?
+
+Open an issue pointing out the successful pull requests you have initiated that have been merged into master, and requesting to be added to @conda-forge/staged-recipes .
```
Comment (bgruening) [inline README.md]: Should we also mention a few reviews that has be done?

**Comment by smithsp** [comment]
If the minimum number of successful pull requests and reviews were set, it seems like there could be a bot to check, for instance,
https://github.com/conda-forge/staged-recipes/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Asmithsp+is%3Amerged
https://github.com/conda-forge/staged-recipes/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+reviewed-by%3Asmithsp
and open an issue to ask the contributor if they would like to join the team.  (I am not familiar with the various GitHub integrations, so I might be proposing something that is not doable or desirable.)

**Comment by smithsp** [comment]
Link to rendered README: https://github.com/smithsp/staged-recipes/tree/join_team

**Comment by jakirkham** [comment]
I don’t think users should be requesting to join staged-recipes through any public medium. It potentially makes the discussion public, which could squash contrary opinions amongst those deciding. Not to mention if the outcome is negative, this could be embarrassing for the person who requested. Thus far we have used private Gitter messages and email to discuss these. Would be open to other private communications. I think @pelson created some emails for the org and some mailing lists. Maybe these could be used.

The content itself seems fine. Though a bit more formal than anything we have done previously.

**Comment by pelson** [comment]
I agree with @jakirkham's sentiment. We want to be very open and welcoming of new contributors, but don't want to be put in the position where we might be making very public rejections (FWIW so far rejections have never been outright - they have been a case of "let's watch X as they are making very positive progress, and could soon reach the core developer bar").

**Comment by patricksnape** [comment]
For what it's worth I agree with @pelson and @jakirkham that we have to be careful about public discussions for being included in the staged-recipes team. If we do have a formal bar that needs to be met then we also need a formal process for how we reach a consensus about including an individual.

**Comment by smithsp** [comment]
I get the impression that I am underestimating the role of the singular mention of gitter in the README:
image
Perhaps the README just needs to say: "Interested in joining `@conda-forge/staged-recipes`?  Review some open pull requests then come contribute to the conversation at gitter.`

**Pull request closed without merging**